docs/usage/config-presets.md

```diff
@@ -38,20 +38,33 @@ In order to achieve these goals, preset configs allow for a very modular approac
+| GitHub default with a tag          | `github>abc/foo#1.2.5`           | `default` | `https://github.com/abc/foo`         | `default.json`  | `1.2.5.`       |
+| GitHub with preset name with a tag | `github>abc/foo:xyz#1.2.5`       | `xyz`     | `https://github.com/abc/foo`         | `xyz.json`      | `1.2.5.`       |
+| GitHub with preset path with a tag | `github>abc/foo//path/xyz#1.2.5` | `xyz`     | `https://github.com/abc/foo`         | `path/xyz.json` | `1.2.5.`       |
+| GitLab default                     | `gitlab>abc/foo`                 | `default` | `https://gitlab.com/abc/foo`         | `default.json`  | Default branch |
+| GitLab with preset name            | `gitlab>abc/foo:xyz`             | `xyz`     | `https://gitlab.com/abc/foo`         | `xyz.json`      | Default branch |
+| GitLab with preset path            | `gitlab>abc/foo//path/xyz`       | `xyz`     | `https://gitlab.com/abc/foo`         | `path/xyz.json` | Default branch |
+| GitLab default with a tag          | `gitlab>abc/foo#1.2.5`           | `default` | `https://gitlab.com/abc/foo`         | `default.json`  | `1.2.5.`       |
+| GitLab with preset name with a tag | `gitlab>abc/foo:xyz#1.2.5`       | `xyz`     | `https://gitlab.com/abc/foo`         | `xyz.json`      | `1.2.5.`       |
+| GitLab with preset path with a tag | `gitlab>abc/foo//path/xyz#1.2.5` | `xyz`     | `https://gitlab.com/abc/foo`         | `path/xyz.json` | `1.2.5.`       |
+| Gitea default                      | `gitea>abc/foo`                  | `default` | `https://gitea.com/abc/foo`          | `default.json`  | Default branch |
+| Gitea with preset name             | `gitea>abc/foo:xyz`              | `xyz`     | `https://gitea.com/abc/foo`          | `xyz.json`      | Default branch |
+| Gitea default with a tag           | `gitea>abc/foo#1.2.5`            | `default` | `https://gitea.com/abc/foo`          | `default.json`  | `1.2.5.`       |
+| Gitea with preset name with a tag  | `gitea>abc/foo:xyz#1.2.5`        | `xyz`     | `https://gitea.com/abc/foo`          | `xyz.json`      | `1.2.5.`       |
+| Local default                      | `local>abc/foo`                  | `default` | `https://github.company.com/abc/foo` | `default.json`  | Default branch |
+| Local with preset path             | `local>abc/foo//path/xyz`        | `default` | `https://github.company.com/abc/foo` | `path/xyz.json` | Default branch |
+| Local default with a tag           | `local>abc/foo#1.2.5`            | `default` | `https://github.company.com/abc/foo` | `default.json`  | `1.2.5.`       |
+| Local with preset path with a tag  | `local>abc/foo//path/xyz#1.2.5`  | `default` | `https://github.company.com/abc/foo` | `path/xyz.json` | `1.2.5.`       |
 
 Note that you can't combine the path and sub-preset syntaxes (i.e. anything in the form `provider>owner/repo//path/to/file:subsubpreset`) is not supported. One workaround is to use distinct files instead of sub-presets.
```

Comment:
```suggestion
Note that you can't combine the path and sub-preset syntaxes (i.e. anything in the form `provider>owner/repo//path/to/file:subsubpreset`) is not supported.
One workaround is to use distinct files instead of sub-presets.
```

Let's use one sentence per line here as well. I know this is not a line that this PR is adjusting, but we can fix it in one go here. 😉